pricing/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Pricing</title>
+		<link rel="stylesheet" href="style.css" />
+	</head>
+	<body>
+		<header>
+			<nav class="pages">
+				<ul>
+					<li><a href="index.html">Home</a></li>
+					<li><a href="about.html">About</a></li>
+					<li><a href="/pricing/index.html">Pricing</a></li>
+				</ul>
+			</nav>
+			<nav class="pages">
+				<ul>
+					<li><a href="login/index.html">Log In</a></li>
+					<li><a href="login/index.html">Sign Up</a></li>
+					<li>
+						<a href="profile.html">
+							<img src="/images/profileIcon.png" alt="Profile" width="18" height="=18" />
+							Profile
+						</a>
+					</li>
+				</ul>
+			</nav>
+		</header>
+		<main>
+			<section id="header">
+				<div id="header-text">
```

Comment:
Not sure if this nested `<div>` is needed

Reply:
Yeah. Seems like you can remove this if you add the `font-size: 4vmin;` rule to the `#header` instead.